cocos/platform/android/javaactivity-android.cpp

```diff
@@ -115,6 +115,23 @@ JNIEXPORT void Java_org_cocos2dx_lib_Cocos2dxRenderer_nativeInit(JNIEnv*  env, j
     cocos2d::network::_preloadJavaDownloaderClass();
 }
 
+JNIEXPORT void JNICALL Java_org_axmol_lib_AxmolRenderer_nativeOnContextLost(JNIEnv*, jclass)
```

Comment:
@rampr shouldn't this be Java_org_cocos2dx_lib_Cocos2dxRenderer_nativeOnContextLost?

Reply:
you are right, will fix @abhinavroy-hc

Reply:
@rampr also Cocos2dxRenderer instead of AxmolRenderer

Reply:
Thanks @abhinavroy-hc. Fixed that also